### PR TITLE
Performance tests for Python get/set sequence

### PR DESCRIPTION
**Pull request by samuelraeburn**

### CODE REVIEW

File: test/python/test_rticonnextdds_performance.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        the_output = one_use_connector.get_output("MyPublisher::PerformanceTestWriter")
+
+        # Wait for discovery between the entities
+        the_input.wait_for_publications(5000)
+        the_output.wait_for_subscriptions(5000)
+
+        # Approach 1.
+        # Set each element of the sequence separately
+        total_time = 0
+        for i in range(0, num_iterations):
```
Comment (hman523) [inline test/python/test_rticonnextdds_performance.py:48]: Do we want to make it easy to run this a different number of times?

File: test/python/test_rticonnextdds_performance.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+            start_time = time.time()
+            for i in range (0, 600000):
+                the_output.instance['myOctSeq[%d]' % (i)] = 2
+            total_time += (time.time() - start_time)
+        average_time = total_time / num_iterations
+        print("Average time setting element-by-element: " + str(average_time))
+        # Average time setting element-by-element: 2.9646800351142883
+
+
+        # Approach 2.
```
Comment (hman523) [inline test/python/test_rticonnextdds_performance.py:58]: Do we want to separate this somehow so if needed we could run one test or the other?

**Comment by hman523** [comment]
Looks great, I put my suggestions as comments on the commit!

File: test/python/test_rticonnextdds_performance.py (hunk 1 above)
Comment (samuelraeburn) [reply test/python/test_rticonnextdds_performance.py:48]: You can currently set the num_iterations fixture to do that, do you have a better way of doing it?

**Comment by samuelraeburn** [comment]
```
sam@rti-10678:~/working/trees/connector/connector-py$ time python3 -m pytest ./test/python/test_rticonnextdds_performance.py 
======================================== test session starts =========================================
platform linux -- Python 3.7.3, pytest-5.0.1, py-1.8.0, pluggy-0.12.0
rootdir: /home/sam/working/trees/connector/connector-py
collected 3 items                                                                                    

test/python/test_rticonnextdds_performance.py ...                                              [100%]

===================================== 3 passed in 966.61 seconds =====================================

real	16m6.818s
user	16m4.668s
sys	0m0.920s
```

File: test/python/test_rticonnextdds_performance.py [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+# This code contains trade secrets of Real-Time Innovations, Inc.             #
+###############################################################################
+
+import pytest,sys,os
+sys.path.append(os.path.dirname(os.path.realpath(__file__))+ "/../../")
+import rticonnextdds_connector as rti
+from test_utils import *
+
+# These tests currently take around 25 minutes to run with 100 repeats.
+# we do. They were added to help verify performance of anothe product pyrti API.
```
Comment (alexcamposruiz) [inline test/python/test_rticonnextdds_performance.py:15]:
```suggestion
# They were added to help verify performance of another product, connextdds-py.
```

File: test/python/test_rticonnextdds_performance.py (hunk 1 above)
Comment (hman523) [reply test/python/test_rticonnextdds_performance.py:43]: Could a command line argument be passed with pytest? If not, just changing num_iterations is fine

**Pull request merged**